website/source/docs/agent/kv.html.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: "docs"
+page_title: "Consul KV"
+sidebar_current: "docs-agent-kv"
+description: |- 
+    Consul KV is a core feature of Consul and is installed with the Consul agent.
+---
+
+# Consul KV 
+
+Consul KV is a core feature of Consul and is installed with the Consul agent. Once installed with the agent, it will have sane defaults. Consul KV allows users to store indexed objects, its main uses are storing configuration parameters and metadata. It is a simple KV store and is not intended to be a full featured datastore, such as DynamoDB, but has some similarities. 
```

Comment:
```suggestion
Consul KV is a core feature of Consul and is installed with the Consul agent. Once installed with the agent, it will have sane defaults. Consul KV allows users to store indexed objects, though its main uses are storing configuration parameters and metadata. Please note that it is a simple KV store and is not intended to be a full featured datastore (such as DynamoDB) but has some similarities to one. 
```